### PR TITLE
ESLint plugin

### PR DESCRIPTION
**Pull request by xtuc**
This is a first try, we can check a lot more.

I'm targeting usage in Webpack and Rollup.

See README https://github.com/xtuc/webassemblyjs/blob/feat-eslint-plugin/packages/eslint/README.md#supported-rules

### CODE REVIEW

File: packages/eslint/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+```
+
+
+Then configure the rules you want to use under the rules section.
+
+```json
+{
+    "rules": {
+        "webassembly/non-unexisting-export": 2
```
Comment (ColinEberhardt) [inline packages/eslint/README.md]: Should that be `no-unexisting-export`

File: packages/eslint/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    }
+}
+```
+
+## Supported Rules
+
+### `no-unexisting-export`
```
Comment (ColinEberhardt) [inline packages/eslint/README.md]: I must admit, I'm not keen on that naming. How about `no-missing-exports`? or `no-exports-missing`?

File: packages/eslint/src/rules/no-unexisting-export.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+        const dirname = path.dirname(context.getFilename());
+
+        const source = object.arguments[0].value;
+        const file = path.join(dirname, source);
+
+        if (existsSync(file) === false) {
+          return context.report({
+            node: object,
+            message: "WASM file does not exists"
```
Comment (ColinEberhardt) [inline packages/eslint/src/rules/no-unexisting-export.js]: `WASM file does not exist`

File: packages/eslint/README.md (hunk 2 above)
Comment (xtuc) [reply packages/eslint/README.md]: I agree, what about `no-unknown-export`?

**Comment by ColinEberhardt** [comment]
nice idea!

File: packages/eslint/README.md (hunk 2 above)
Comment (ColinEberhardt) [reply packages/eslint/README.md]: yeah, `no-unknown-export` sounds good too

**Comment by xtuc** [comment]
Thanks for your review @ColinEberhardt, I updated this.

**Pull request merged**